easybuild/easyconfigs/s/SciPy-bundle/SciPy-bundle-2024.05-gfbf-2023a.eb

```diff
@@ -0,0 +1,102 @@
+easyblock = 'PythonBundle'
+
+name = 'SciPy-bundle'
+version = '2024.05'
```

Comment:
@verdurin I'm not a big fan of introducing another `SciPy-bundle` version in the `2023a` generation of easyconfigs (it will lead to confusion for sure).

Is that really the most viable approach here?
Which bits of the existing `SciPy-bundle` version are not recent enough?
Is using a more recent toolchain for all this not a better option?

I won't strictly oppose it, but this effectively makes anything that requires this more recent `ScIPy-bundle` version incompatible with anything else that requires the "common" `SciPy-bundle` version when a `2023a` toolchain is used.

Reply:
Have reverted to the default `SciPy-bundle` for this toolchain.